### PR TITLE
feat: add icon for dark and light theme

### LINKED ISSUE #528: Add icon for clarity
## Summary
Add icons to the dark mode switch tab

### What's new?
It adds an icon for light mode and dark mode to the existing component to show it present status

### Motivation
when I first got to the website, I didn't understand what the switch was for until I clicked it, not a good user experience

### Additional context
image

### PR DESCRIPTION
**Pull request by Dun-sin**
## Changes proposed
- Add icons ☀️, 🌜 for light and dark mode respectively

## Screenshot
image

### CODE REVIEW

**Comment by kartikcho** [comment]
@Dun-sin is it possible to use emojis using the existing `react-easy-emoji` dependency? This way the emojis would be consistent across platforms.

Also minor nit but there's a very little 1px space in day mode slider state (see gif below). Making the switch go slightly extra behind in day mode slider state should fix it (similar to night mode state)

Screen Cast 2022-10-01 at 9 37 24 PM

**Comment by Dun-sin** [comment]
I don't think there's a way to style a before element in react JSX, i looked for a solution and couldn't find one, maybe I can keep trying to find a solution, if you want. 

Also the 1px space isn't part of my PR, if you checked my files I never touched anything else, maybe it should be a separate issue if you want a fix for it

**Comment by kartikcho** [comment]
> I don't think there's a way to style a before element in react JSX

Have you tried setting the content in `style={{}}`? This should be a solved problem I think

> Also the 1px space isn't part of my PR

Yeah I know it isn't a part of your PR but we can fix that with this one too. If you want to create a separate issue and a PR for it, that's alright, just add me as the reviewer so I dont miss it.

**Comment by Dun-sin** [comment]
> > I don't think there's a way to style a before element in react JSX
> 
> Have you tried setting the content in `style={{}}`? This should be a solved problem I think
> 
> > Also the 1px space isn't part of my PR
> 
> Yeah I know it isn't a part of your PR but we can fix that with this one too. If you want to create a separate issue and a PR for it, that's alright, just add me as the reviewer so I dont miss it.

Yeah I know about the style, but I need to use before pseudoclass, and I don't think to can assess it that way.

I would rather create a new issue

**Comment by Dun-sin** [comment]
@kartikcho what do you think, should I leave it or try to still figure it out

**Comment by kartikcho** [comment]
@Dun-sin I did a quick search and it looks like it is possible with two solutions that you could try out. These methods are also new to me so I'm excited to see this work!

- [Using a variable for the content in CSS](https://stackoverflow.com/a/65690431)
- [Create a new pseudo element via react](https://stackoverflow.com/a/28269950)

**Comment by Dun-sin** [comment]
> @Dun-sin I did a quick search and it looks like it is possible with two solutions that you could try out. These methods are also new to me so I'm excited to see this work!
> 
>     * [Using a variable for the content in CSS](https://stackoverflow.com/a/65690431)
> 
>     * [Create a new pseudo element via react](https://stackoverflow.com/a/28269950)

I saw the second resource, but I didn't want to create a new element

**Review by Dun-sin — COMMENTED** [review]
Seems fine now, let me know what you think

File: src/components/ToggleSwitch/ToggleSwitch.js [hunk 1]
```diff
@@ -7,6 +8,16 @@ const ToggleSwitch = () => {
...
+  const emojis = {
+    light: emoji('☀️'), 
+    dark: emoji('🌜')
+  }
+  const themeStyle = {
+    "--light-mode": emojis.light,
+    "--dark-mode": emojis.dark
+  }
+
+  console.log(isChecked)
```
Comment (kartikcho) [inline src/components/ToggleSwitch/ToggleSwitch.js]: leftover log

**Review by kartikcho — CHANGES_REQUESTED** [review]

File: src/components/ToggleSwitch/ToggleSwitch.js [hunk 2]
```diff
@@ -17,7 +28,9 @@ const ToggleSwitch = () => {
           setChecked(!isChecked);
         }}
       />
-      <span className="slider round"></span>
+      <span className="slider round">
+        <span className="emoji">{isChecked ? themeStyle["--dark-mode"] : themeStyle["--light-mode"]}</span>
```
[the remark this answers is not in the document]
Comment (Dun-sin) [reply src/components/ToggleSwitch/ToggleSwitch.js]: Trying to keep a clean code, but I can take it off if you want

**Review by Dun-sin — COMMENTED** [review]
done as you asked, tell me what you think

File: src/components/ToggleSwitch/ToggleSwitch.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (kartikcho) [reply src/components/ToggleSwitch/ToggleSwitch.js]: I don't see how creating 2 abstractions for a simple conditional is clean code. Thanks for simplifying it

**Review by kartikcho — APPROVED** [review]
Looks good!

**Comment by kartikcho** [comment]
@allcontributors please add @Dun-sin for code

**Pull request merged**